src/client/gl-base-technique.cc

```diff
@@ -139,6 +157,12 @@ GlBaseTechnique::Bind(std::weak_ptr<GlVertexArray> vertex_array)
   pending_.vertex_array_damaged = true;
 }
 
+void
+GlBaseTechnique::Bind(std::weak_ptr<GlTexture> texture, uint32_t target)
+{
+  pending_.gl_textures.emplace_back(std::make_pair(target, texture));
```

Comment:
It is not actually emplacing, use push_back for clarity.

```suggestion
  pending_.gl_textures.push_back(std::make_pair(target, texture));
```